api/src/resources/comment.rs

```diff
@@ -418,8 +418,8 @@ impl AnnotatedComment {
 pub struct ThreadProperties {
     duration: Option<NotNan<f64>>,
     response_time: Option<NotNan<f64>>,
-    num_messages: u64,
-    thread_position: u64,
+    num_messages: Option<u64>,
```

Comment:
```suggestion
    num_messages: u64,
```

Reply:
As discussed, `num_messages` is always present if thread properties are present at all.